build.gradle

```diff
@@ -353,6 +353,21 @@ task sparkJar(type: ShadowJar) {
     classifier = 'spark'
 }
 
+task distArchive(type: Zip) {
+    dependsOn shadowJar, sparkJar
+
+    baseName = project.name + "-" + project.version
+    destinationDir file("$buildDir")
+    archiveName baseName + ".zip"
+
+    from(shadowJar.archivePath)
+    from(sparkJar.archivePath)
+    from("settings.gradle")
+    from("gatk-launch")
+    from("README.md")
```

Comment:
you could add something like: 
```
    doFirst {
        assert file'gatk-launch").exists()
        assert file("settings.gradle").exists()
        assert file("README.md").exists()
        ...
    }
```

I don't think it's necessary to check that the archives exist since their failure to exist would have been a task failure on the earlier tasks.  It might be worth doing for the other files though.